src/content/en/2021/privacy.md

```diff
@@ -5,572 +5,541 @@ description: Privacy chapter of the 2021 Web Almanac covering adoption and impac
 
 ### IAB's Consent Frameworks
 
-The <a hreflang="en" href="https://iabeurope.eu/transparency-consent-framework/">Transparency and Consent Framework</a> (TCF) is an initiative of the Interactive Advertising Bureau Europe for providing an industry standard for communicating user consent to advertisers. The framework consists of a Global Vendor List, in which vendors can specify the legitimate purpose of the processed data, and a list of CMPs who act as an intermediary between the vendors and the publishers. Each CMP is responsible for communicating the legal basis and storing the consent option provided by the user in the browser. We refer to the stored cookie as the "consent string".
-
-TCF is meant as a GDPR-compliant mechanism in Europe, although [a recent decision by the Belgian Data Protection Authority](FIXME # https://iabeurope.eu/all-news/update-on-the-belgian-data-protection-authoritys-investigation-of-iab-europe/ or https://www.iccl.ie/news/online-consent-pop-ups-used-by-google-and-other-tech-firms-declared-illegal/) found that this system is still infringing. When the CCPA came into play in California, IAB Tech Lab US developed a <a hreflang="en" href="https://iabtechlab.com/standards/ccpa/">CCPA Compliance Framework for Publishers & Technology Companies</a> (USP), using the same concepts.
-
-Below, we show the distribution of the usage of both versions of TCF and of USP. Note that the crawl is US-based, therefore we do not expect many websites to have implemented TCF. Fewer than 2% of websites use any TCF version, while twice as many websites use the US Privacy framework.
+The <a hreflang="en" href="https://iabeurope.eu/transparency-consent-framework/">Transparency and Consent Framework</a> (TCF) is an initiative of the Interactive Advertising Bureau Europe (IAB) for providing an industry standard for communicating user consent to advertisers. The framework consists of a <a hreflang="en" href="https://iabeurope.eu/vendor-list/">Global Vendor List</a>, in which vendors can specify the legitimate purpose of the processed data, and a list of CMPs who act as an intermediary between the vendors and the publishers. Each CMP is responsible for communicating the legal basis and storing the consent option provided by the user in the browser. We refer to the stored cookie as the _consent string_.
 
+TCF is meant as a GDPR-compliant mechanism in Europe, although [a recent decision by the Belgian Data Protection Authority](https://iabeurope.eu/all-news/update-on-the-belgian-data-protection-authoritys-investigation-of-iab-europe/){# TODO decide if using that URL or https://www.iccl.ie/news/online-consent-pop-ups-used-by-google-and-other-tech-firms-declared-illegal/ #} found that this system is still infringing. When the CCPA came into play in California, IAB Tech Lab US developed a <a hreflang="en" href="https://iabtechlab.com/standards/ccpa/">CCPA Compliance Framework for Publishers & Technology Companies</a> (USP), using the same concepts.
```

Comment:
This was set to two URLs with a TODO to decide, but looks like you forgot to decide. I picked the first and put the other in a comment. Can you review?